content/en/docs/contribute/participating.md

```diff
@@ -44,22 +44,22 @@ aspects of Kubernetes -- the Kubernetes website and documentation.
   commitment to the project. An approver can merge pull requests
   and publish content on behalf of the Kubernetes organization.
   Approvers can also represent SIG Docs in the larger Kubernetes community.
-  Some of the duties of a SIG Docs approver, such as coordinating a release,
+  Some duties of a SIG Docs approver, such as coordinating a release,
   require a significant time commitment.
 
 ## Anyone
 
 Anyone can do the following:
 
-- File a GitHub issue against any part of Kubernetes, including documentation.
+- Open a GitHub issue against any part of Kubernetes, including documentation.
 - Provide non-binding feedback on a pull request
 - Bring up ideas for improvement on [Slack](http://slack.k8s.io/) or the [SIG docs mailing list](https://groups.google.com/forum/#!forum/kubernetes-sig-docs)
-- Use the `/lgtm` Prow command (short for "looks good to me") to show that you're happy with the changes in a pull request.
+- Use the `/lgtm` Prow command (short for "looks good to me") to show your approval of changes in a pull request.
```

Comment:
Now that I think about it, using "approval" to describe something that isn't `/approve` seems fraught. Maybe "affirm"? 
```suggestion
- Use the `/lgtm` Prow command (short for "looks good to me") to affirm the changes in a pull request.
```

Reply:
"Affirm" is a little confusing in this context. I'm going to try the following:

```
- Use the `/lgtm` Prow command (short for "looks good to me") to recommend the changes in a pull request for merging.
```